### PR TITLE
V2.1.2 branch

### LINKED ISSUE #1563: Branding does not show up correctly on grubmenu with kairos 2.1.1
```
**Kairos version:**
root@edge-89a33842be829ddebe91aba70105bcfc:~# cat /etc/os-release
PRETTY_NAME="Ubuntu 22.04.2 LTS"
NAME="Ubuntu"
VERSION_ID="22.04"
VERSION="22.04.2 LTS (Jammy Jellyfish)"
VERSION_CODENAME=jammy
ID=ubuntu
ID_LIKE=debian
HOME_URL="https://www.ubuntu.com/"
SUPPORT_URL="https://help.ubuntu.com/"
BUG_REPORT_URL="https://bugs.launchpad.net/ubuntu/"
PRIVACY_POLICY_URL="https://www.ubuntu.com/legal/terms-and-policies/privacy-policy"
UBUNTU_CODENAME=jammy
KAIROS_NAME="kairos-core-ubuntu-22-lts"
KAIROS_VERSION="v2.1.1"
KAIROS_ID="kairos"
KAIROS_ID_LIKE="kairos-core-ubuntu-22-lts"
KAIROS_VERSION_ID="v2.1.1"
KAIROS_PRETTY_NAME="kairos-core-ubuntu-22-lts v2.1.1"
KAIROS_BUG_REPORT_URL="https://github.com/kairos-io/kairos/issues"
KAIROS_HOME_URL="https://github.com/kairos-io/kairos"
KAIROS_IMAGE_REPO="quay.io/kairos/core-ubuntu-22-lts"
KAIROS_IMAGE_LABEL="latest"
KAIROS_GITHUB_REPO="kairos-io/kairos"
KAIROS_VARIANT="core"
KAIROS_FLAVOR="ubuntu-22-lts"KAIROS_NAME="core-ubuntu-22-lts-k3s:v3.4.7-alpha2"
KAIROS_VERSION="1.24.6-k3s1_v3.4.7-alpha2"
KAIROS_ID="core-ubuntu-22-lts-k3s"
KAIROS_ID_LIKE="core-ubuntu-22-lts-k3s:v3.4.7-alpha2"
KAIROS_VERSION_ID="1.24.6-k3s1_v3.4.7-alpha2"
KAIROS_PRETTY_NAME="core-ubuntu-22-lts-k3s:v3.4.7-alpha2 1.24.6-k3s1_v3.4.7-alpha2"
KAIROS_BUG_REPORT_URL=""
KAIROS_HOME_URL=""
KAIROS_IMAGE_REPO="gcr.io/spectro-dev-public/aslam"
KAIROS_IMAGE_LABEL="v2.1.1_1.24.6-k3s1_v3.4.7-alpha2"
KAIROS_GITHUB_REPO=""
KAIROS_VARIANT="
```

**CPU architecture, OS, and Version:**
`Linux edge-89a33842be829ddebe91aba70105bcfc 5.19.0-45-generic #46~22.04.1-Ubuntu SMP PREEMPT_DYNAMIC Wed Jun 7 15:06:04 UTC 20 x86_64 x86_64 x86_64 GNU/Linux`

**Describe the bug**
Branding does not show up fine on the grubmenu. Please find the attached screenshot.
Kairos 2.1.1
Screenshot 2023-06-27 at 2 30 35 PM (2)

Kairos 2.2.0
Screenshot 2023-06-28 at 8 12 28 PM

**To Reproduce**

**Expected behavior**
Branding to show up correctly on the grubmenu.

**Logs**

**Additional context**

### PR DESCRIPTION
**Pull request by mauromorales**
**What this PR does / why we need it**:

Backporting of https://github.com/kairos-io/kairos/commit/afd650e5db3f9048e006e277212877579096d519 into v2.1.1

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #1563

### CODE REVIEW

File: overlay/files/system/oem/08_grub.yaml [hunk 1]
```diff
@@ -102,7 +102,7 @@ stages:
       name: "Grub branding"
       if: '[ -e "/etc/kairos/branding/grubmenu.cfg" ]'
       commands:
-      - cp -rfv /etc/kairos/branding/grubmenu.cfg /tmp/mnt/STATE/grubmenu
+        - cp -rfv /etc/kairos/branding/grubmenu.cfg /tmp/mnt/STATE/grubmenu
```
Comment (mudler) [inline overlay/files/system/oem/08_grub.yaml:105]: this doesn't look right :thinking:

Comment (mauromorales) [reply overlay/files/system/oem/08_grub.yaml:105]: @mudler as in, it looks different from https://github.com/kairos-io/kairos/pull/1426/files or as in, maybe that was not the actual fix?

Comment (mudler) [reply overlay/files/system/oem/08_grub.yaml:105]: the diff shows only a whitespace, only on the branch.. should look a bit different?

Comment (mauromorales) [reply overlay/files/system/oem/08_grub.yaml:105]: LOL I see what you mean ... it's already there ... let me check if I based it from teh right one :/

**Pull request closed without merging**